io/net/virt-net/network_virtualization.py

```diff
@@ -113,7 +113,8 @@ def setUp(self):
         if 'vios' in str(self.name.name):
             self.session = Session(self.vios_ip, user=self.vios_user,
                                    password=self.vios_pwd)
-            self.session.connect()
+            if not self.session.connect():
+                self.fail("Failed connecting to VIOS")
```

Comment:
@cforno12 Only one test (`test_rmdev_viosfailover`) uses VIOS login, so only that test should fail.

We should move this part of the code inside the test.

Reply:
Right, but there could be more tests in the future that use a VIOS connection, therefore we should check if the connection is successful when setting up.

Reply:
@cforno12 What I meant is, we can push the vios session setup itself into those tests that use it. Else, we would end up failing tests that do not use a vios session at all.

Can you include that change in your patch please ?

Reply:
Sure, thanks Narasimhan